mllib/src/test/scala/org/apache/spark/ml/classification/GBTClassifierSuite.scala

```diff
@@ -223,15 +278,18 @@ private object GBTClassifierSuite extends SparkFunSuite {
   /**
    * Train 2 models on the given dataset, one using the old API and one using the new API.
    * Convert the old model to the new format, compare them, and fail if they are not exactly equal.
+   *
+   * The old API only supports variance-based impurity, so gbt should have that setting.
    */
   def compareAPIs(
       data: RDD[LabeledPoint],
       validationData: Option[RDD[LabeledPoint]],
       gbt: GBTClassifier,
       categoricalFeatures: Map[Int, Int]): Unit = {
+    assert(gbt.getImpurity == "variance")
```

Comment:
Use triple equals ```===``` in test suites for comparisons.  It doesn't matter here, but I like encouraging it since otherwise people forget to do it when they should.